packages/framework/src/Model/Feed/Standard/StandardFeedItemRepository.php

```diff
@@ -51,7 +49,6 @@ public function getItems(DomainConfigInterface $domainConfig, $seekItemId, $maxR
             ->addSelect('a')->join('p.calculatedAvailability', 'a')
             ->addSelect('b')->leftJoin('p.brand', 'b')
             ->andWhere('p.variantType != :variantTypeMain')->setParameter('variantTypeMain', Product::VARIANT_TYPE_MAIN)
-            ->join(ProductDomain::class, 'pd', Join::WITH, 'pd.product = p.id AND pd.domainId = :domainId')
```

Comment:
Well since ProductDomain was not property of Product, there was need to join in query, or i dont understand the question right. But you are right, i added it there.